docs/configuration/index.md

```diff
@@ -5,6 +5,7 @@ Read about the various ways to configure uv:
 - [Using configuration files](./files.md)
 - [Using environment variables](./environment.md)
 - [Configuring authentication](./authentication.md)
+- [Configuring package indexes](./indexes.md)
```

Comment:
Just wondering, why here instead of in `Concepts`?

Reply:
I'm not sure. To me it was similar to authentication.